Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -217,6 +217,27 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
       .map(cookieFrom(refTag:project:))
       .skipNil()
       .observeValues { AppEnvironment.current.cookieStorage.setCookie($0) }
+
+    let shouldTrackCTATappedEvent = ctaButtonTappedWithType
+      .filter { [.pledge, .seeTheRewards, .viewTheRewards].contains($0) }
```

Comment:
We weren't actually triggering the event on the experimental variants. Added the here and added tests to make sure we're tracking taps to the button in all three cases.

Reply:
Do you think we should do this or just filter on the project not being backed yet? That way we don't need to update this when variants change 🤔

Reply:
I think we need to be explicit about when we track an "impression" for an experimental variant being acted on. If we don't know for sure that a variant has been seen/activated, then we shouldn't track an event in Optimizely.